### ISSUE TITLE
Investigate the integration with nodejs 12.13

### ISSUE THREAD

**Issue by licanhua** [microsoft/react-native-windows#3611]
All of our testing and setup are based on nodejs 10, but now nodejs stable version is 12.13.0.
We experienced error with 12.13 before. Although problem is not in our code, but we are impacted and can't start the metro bundle server.

Here is the error
>yarn run start
$ react-native start
error Invalid regular expression: /(.*\\__fixtures__\\.*|node_modules[\\\]react[\\\]dist[\\\].*|website\\node_modules\\.*|heapCapture\\bundle\.js|.*\\__tests__\\.*)$/: Unterminated character class. Run CLI with --verbose flag for more details.
SyntaxError: Invalid regular expression: /(.*\\__fixtures__\\.*|node_modules[\\\]react[\\\]dist[\\\].*|website\\node_modules\\.*|heapCapture\\bundle\.js|.*\\__tests__\\.*)$/: Unterminated character class
    at new RegExp (<anonymous>)
    at blacklist (f:\repo\react-native-windows\node_modules\metro-config\src\defaults\blacklist.js:34:10)
    at getBlacklistRE (f:\repo\react-native-windows\node_modules\@react-native-community\cli\build\tools\loadMetroConfig.js:69:59)
    at getDefaultConfig (f:\repo\react-native-windows\node_modules\@react-native-community\cli\build\tools\loadMetroConfig.js:84:20)
    at load (f:\repo\react-native-windows\node_modules\@react-native-community\cli\build\tools\loadMetroConfig.js:112:25)
    at Object.runServer [as func] (f:\repo\react-native-windows\node_modules\@react-native-community\cli\build\commands\server\runServer.js:82:58)
    at Command.handleAction (f:\repo\react-native-windows\node_modules\@react-native-community\cli\build\cliEntry.js:160:21)
    at Command.listener (f:\repo\react-native-windows\node_modules\commander\index.js:315:8)
    at Command.emit (events.js:210:5)
    at Command.parseArgs (f:\repo\react-native-windows\node_modules\commander\index.js:651:12)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by acoates-ms** [comment]
Node 12 is the current TLS version of node -- this basically means that RNW is unusable for the vast majority of web/JS developers.

**Comment by chrisglein** [comment]
@licanhua You mentioned elsewhere that there was a community issue/discussion on this. Can you link to that?

Workaround is to use an earlier version, which the docs already point to. Not ideal, as people are installing Node for more than just this.

Is this broken in the CLI? Is the community addressing this with 61 or something?

**Comment by licanhua** [comment]
https://github.com/facebook/react-native/issues/26598
And there is a PR https://github.com/facebook/metro/pull/464 on metro.

**Comment by licanhua** [comment]
metro fixed it on 0.57.0 in PR https://github.com/facebook/metro/pull/468.
@react-native-community (latest 3.0.4)  is still use 0.56. and it's not compatible with new metro-config:
```
$ react-native start
error (0 , _metro(...).createBlacklist) is not a function. Run CLI with --verbose flag for more details.
TypeError: (0 , _metro(...).createBlacklist) is not a function
    at getBlacklistRE (D:\repo\react-native-windows\node_modules\@react-native-community\cli\build\tools\loadMetroConfig.js:66:59)
    at getDefaultConfig (D:\repo\react-native-windows\node_modules\@react-native-community\cli\build\tools\loadMetroConfig.js:82:20)
    at load (D:\repo\react-native-windows\node_modules\@react-native-community\cli\build\tools\loadMetroConfig.js:118:25)
    at Object.runServer [as func] (D:\repo\react-native-windows\node_modules\@react-native-community\cli\build\commands\server\runServer.js:82:58)
    at Command.handleAction (D:\repo\react-native-windows\node_modules\@react-native-community\cli\build\index.js:164:23)
```

We can't resolve this until @react-native-community fixed https://github.com/react-native-community/cli/pull/864
